internal/common/mqtt/mqttconfig.go

```diff
@@ -70,6 +72,11 @@ func SetClientID(id string) Config {
 	}
 }
 
+// GetClientID sets the mqtt client id.
+func GetClientID() string {
+	return mqttClient.ID
+}
+
```

Comment:
Nothing to call this.

Reply:
SInce added setter hence added the getter for future use. Will remove it for now

Reply:
Done